### ISSUE TITLE
Forums are down

### ISSUE THREAD

**Issue by lwcorp** [eegeeZA/spamgourmet-clone#69]
[The forums](https://bbs.spamgourmet.com) have been down for at least half a month ([see if it's still down](https://www.isitdownrightnow.com/bbs.spamgourmet.com.html)) with [500 internal error](https://en.wikipedia.org/wiki/500_error).

I was told there was a deletion of the relevant VM for some reason, and it's not clear what happened to the backup, but it's under investigation.

Please remember the forums are the only publicly known way to reach anyone at Spamgourmet, even if it's only other users.

Possible archives:
1. https://web.archive.org/web/20231209174104/https://bbs.spamgourmet.com/
1. https://archive.md/SGi40

![image](https://github.com/spamgourmet/spamgourmet/assets/1773306/edc4d1fc-596e-450a-a2ce-c56cc67d64c4)
![image](https://github.com/spamgourmet/spamgourmet/assets/1773306/902bff70-e198-4e39-a3c6-8d6fa7dcd8a7)

**Comment by jgonggrijp** [comment]
Thanks for reporting. @josiah-hamilton is aware of it and will be trying to get the BBS back online this weekend. He already determined that the data have been preserved, the VPS is just not running for some reason. Hang tight!

**Comment by akinet** [comment]
The forum is still down.
And the website certificates for www and bbs are going to expire next Sunday. This will mean that after that date we will not be able to access the main website or the forum even if the current problem is solved. I don't know if the expiration of these certificates will affect the sending of emails.
 Do you need help to resolve these issues?

**Comment by jgonggrijp** [comment]
@akinet Thank you for alerting us and for your generous offer to help. @josiah-hamilton is aware of the expiring certificates and is working on extending them as I'm writing this.

While these certificates are taken care of for now, we can definitely use some help with system administration and development more in general. If you like, please shoot an email to me and I can get you in touch with our private communication channels.

**Comment by lwcorp** [comment]
The forum's certification indeed expired today...(while at least the main site's certificate was extended for another 3 months)
![image](https://github.com/spamgourmet/spamgourmet/assets/1773306/d1ef96b1-0793-4c30-ad79-ea5c12625608)
![image](https://github.com/spamgourmet/spamgourmet/assets/1773306/29e50300-ba0b-4f83-9c0c-292ed1717b35)

**Comment by jgonggrijp** [comment]
@lwcorp The main site now uses letsencrypt en will autorenew from now on. It should be the same for the bbs, but apparently something went wrong. Thanks for reporting!

**Comment by NikoMuffin** [comment]
@jgonggrijp @josiah-hamilton 

Forums are down (again or still, not sure).

**Comment by HenningKlaskala** [comment]
Forums are still (or again?) down.

E-mails to the former SG domain disposableaddress.com don't arrive.
That domain is for sale.
www.spamgourmet.com still supports sending e-mail "from" disposableaddress.com 

It would be nice to announce the deletion of SG domains somewhere as early as possible.

**Comment by jgonggrijp** [comment]
Thanks for reporting, @HenningKlaskala and @NikoMuffin. It appears that the BBS certificate has expired (again?), so that should just be a matter of a renewal. As for the domain, we will be looking into that.

**Comment by lwcorp** [comment]
As far as I've noticed, the forums have never returned once since this issue was submitted, which is why the issue was never un-opened.

**Comment by lwcorp** [comment]
And just like that the forums are back...but disabled. Will that be fixed soon?

![image](https://github.com/user-attachments/assets/3baef3d3-4f47-4db5-b7f4-f0a9d0e98d29)

**Comment by josiah-hamilton** [comment]
Yes. Phpbb is partially migrated to the newer version that makes use of php7, but I need to complete the migration steps when I am off mobile. 
Presently the database exists but is not being referenced properly by the webserver.

**Comment by lwcorp** [comment]
Thanks! Don't you prefer the latest PHP 8 (first released in 2020) while you're at it?
To quote PHP's website itself:

> Support for PHP 7 has been **discontinued** since 03 Nov 2022. Please consider upgrading to 8.

**Comment by lwcorp** [comment]
Exactly half a year later, forums are back, thanks!

**Comment by lwcorp** [comment]
It happened again a few weeks ago, this time due to `Account Suspended`....which sounds like hosting wasn't paid for. Is this where donations could assist? Alternatively, you could switch to a free hosting for example a Discord server.
It's hard to rely on the forums if they keep coming and going like that.

![Image](https://github.com/user-attachments/assets/1d245c0e-48a2-4f00-b41b-37d7b5f14923)

**Comment by sba923** [comment]
Hi folks, what we users (and possibly donators) do about this?

**Comment by jgonggrijp** [comment]
@josiah-hamilton is dealing with personal circumstances that currently prevent him from solving the issue with the forums. In the meanwhile, please use this GitHub issue tracker to report anything that doesn't work about spamgourmet itself (or +1 existing tickets). Sorry for the inconvenience!

The intention is still to bring the forums back to life, so leaving this open in the meanwhile.

**Comment by lwcorp** [comment]
Why not temporarily write just that there?
Keeping the suspension notice for months (1 year soon enough) doesn't look good for this service.

**Comment by jgonggrijp** [comment]
Agreed that this would be better, but the same issue that prevents @josiah-hamilton from bringing the forums back to life is also preventing him from doing anything else about it. For example, it would be better if it didn't all depend on him alone (legacy from his late father), but he currently cannot change that, either. He is dealing with a calamity.

It doesn't look good and it honestly isn't, but that's the way it is right now. Sorry. Donations would help, but patience and understanding help most.

**Comment by Alpha-Hoernchen** [comment]
Sorry to hear that! I tried to donate, but it seems the paypal donation buttons are not working either :-(

**Comment by taxlinda** [comment]
I tried to access the forum via the Spamgourmet site today. 401 errors. Then, I went on a hunt to see what's up and eventually found this forum. 

Is there any news? I'm getting concerned about the viability of the site. I have far too many spamgourmet email addies. I got my first Spamgourmet email address in 2004. I also could not make a donation with PayPal.

**Comment by jgonggrijp** [comment]
No news. I expect the email service and the core website to continue to work (mostly), but PayPal and the forums might still take a long time to resolve.

**Comment by lwcorp** [comment]
I wish it would have just redirected here.

**Comment by kvaradhan3** [comment]
> No news. I expect the email service and the core website to continue to work (mostly), but PayPal and the forums might still take a long time to resolve.

Not sure.   Email service for existing subscribers continues, but new subscribers get `500 Internal Server Error`.

Looks like a great service, wish I could enroll :-(

**Comment by kvaradhan3** [comment]
> > No news. I expect the email service and the core website to continue to work (mostly), but PayPal and the forums might still take a long time to resolve.
> 
> Not sure. Email service for existing subscribers continues, but new subscribers get `500 Internal Server Error`.
> 
> Looks like a great service, wish I could enroll :-(

This particular issue of new users is also in #77 and fixed there.
